openmc/tallies.py

```diff
@@ -452,9 +449,13 @@ def add_score(self, score):
                   'not a string'.format(score, self.id)
             raise ValueError(msg)
 
-        # If the score is already in the Tally, don't add it again
+        # If the score is already in the Tally, raise an error
         if score in self.scores:
-            return
+            msg = 'Unable to add a duplicate score {0} to Tally ID="{1}" ' \
```

Comment:
So only the Python API precludes duplicate scores? I need to check the rest of this PR to find out I suppose.